test_base_time_window/models/res_partner.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from collections import defaultdict
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+
+    _inherit = "res.partner"
+
+    time_window_ids = fields.One2many(
```

Comment:
this field and the method IMO could stay in another mixin `time.window.consumer.mixin`